### PR TITLE
[MIG][16.0] product_packaging_type_pallet => product_packaging_level_pallet

### PR DESCRIPTION
**Pull request by rousseldenis**
Depends on:

- #1215 
- #1220

### CODE REVIEW

File: product_packaging_level_pallet/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_packaging_type_pallet
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/135/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds specific fields to manage product packaging types for pallets.
```
Comment (lmignon) [inline product_packaging_level_pallet/README.rst:28]:
```suggestion
This module adds specific fields to manage product packaging levels for pallets.
```

File: product_packaging_level_pallet/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Product Packaging Type Pallet",
```
Comment (lmignon) [inline product_packaging_level_pallet/__manifest__.py:4]:
```suggestion
    "name": "Product Packaging Level Pallet",
```

File: product_packaging_level_pallet/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "summary": "Manage packaging of pallet type",
```
Comment (lmignon) [inline product_packaging_level_pallet/__manifest__.py:5]:
```suggestion
    "summary": "Manage packaging level of pallet",
```

File: product_packaging_level_pallet/readme/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Project C2C Migration information
```
Comment (lmignon) [inline product_packaging_level_pallet/readme/README.md:1]: This file should be removed

**Review by lmignon — CHANGES_REQUESTED** [review]

**Review by FrancoMaxime — APPROVED** [review]
I agree with @lmignon comments

File: product_packaging_level_pallet/README.rst (hunk 1 above)
Comment (jbaudoux) [reply product_packaging_level_pallet/README.rst:28]: No, don't change this, it is a package type specialization, not a package level

File: product_packaging_level_pallet/__manifest__.py (hunk 2 above)
Comment (jbaudoux) [reply product_packaging_level_pallet/__manifest__.py:4]: No, don't change this, it is a package type specialization, not a package level

File: product_packaging_level_pallet/__manifest__.py (hunk 3 above)
Comment (jbaudoux) [reply product_packaging_level_pallet/__manifest__.py:5]: No, don't change this, it is a package type specialization, not a package level

**Review by jbaudoux — CHANGES_REQUESTED** [review]
Now that we have package level and package types, this needs to be migrated to the right logistic concept.

It is a packaging type that can be marked as a pallet.
It doesn't make sense to mark a package level as a pallet. A pallet is a tertiary level. There is always one and only one tertiary level. However, you can have multiple package type on a tertiary level and some can be marked as pallet.

Rename the module, and remove the dependency on product_packaging_level. Set is_pallet on package type instead.

**Comment by bealdav** [comment]
Hi, thanks for this contribution.
I don't if this PR 'll be achieved considering @jbaudoux comment.

On my side, regarding the python code I don't know why this module depends on product_packaging_dimension. None of the python code is related to this module. Probably only because of the xml view.

To guess what is a pallet, I suppose that level can be used without this module (convention by project ?)

Do you what'll become this PR ? Thanks

**Comment by hspl-priva** [comment]
LGTM @rousseldenis @pedrobaeza tested. it seems good. any idea when can we merge this?

**Review by hspl-priva — APPROVED** [review]

**Comment by simahawk** [comment]
@rousseldenis ping

**Comment by bosd** [comment]
@rousseldenis @jbaudoux Could you attend to this PR and save it from being lost?

**Comment by baguenth** [comment]
Hey @rousseldenis @jbaudoux @lmignon , 

any chance we can get this integrated?

**Comment by jbaudoux** [comment]
@baguenth This module has been abandonned. If you want to recover layers and packs_per_layer on the package type, you can make a new simple module with those 2 fields.

**Pull request closed without merging**